setup.py

```diff
@@ -16,16 +16,37 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 # ----------------------------------------------------------------------------
 
+from importlib.util import spec_from_file_location, module_from_spec
 import os
-import sys
-from metomi.isodatetime import __version__
+from pathlib import Path
 # overriding setuptools command
 # https://stackoverflow.com/a/51294311
 from setuptools.command.bdist_rpm import bdist_rpm as bdist_rpm_original
 # to parse pytest command arguments
 # https://docs.pytest.org/en/2.8.7/goodpractices.html#manual-integration
 from setuptools.command.test import test as TestCommand
 from setuptools import setup, find_namespace_packages
+import sys
+
+
+try:
+    os.chdir(os.path.dirname(__file__))
+except FileNotFoundError:
+    pass
```

Comment:
Currently, if the current working directory is not the same directory containing the library, the default behaviour for something like `python3 /path/to/metomi/isodatetime/setup.py bdist` is to build a distribution file under `$PWD/dist/`.

Does this change the behaviour? (May surprise some usage.)

Reply:
@oliver-sanders This block is still changing the current working directory after your latest commit. Is this still necessary?

Reply:
Forgot to commit the change which removed it.